ruby/vampires.rb

```diff
@@ -28,12 +34,32 @@
 		puts "Please answer yes or no."
 	end
 
+
+puts "Please list your allergies and enter them one at a time. Type 'done' when you are finished."
+allergies = gets.chomp
+	if allergies == "sunshine"
+		sunshine_allergy = true	
+	end
+	loop do 
```

Comment:
`Loop do.. break` is not the best choice for this structure. What you are really trying to say is that you want to loop until a certain condition is met, so might as well use the structure that reflects that. 

`until multiple_allergies == 'done'`